README.md

```diff
@@ -1,12 +1,33 @@
 # ALBA
+This is the rust library of the *Telescope* protocol proposed in the paper [Approximate Lower Bound Arguments](https://iohk.io/en/research/library/papers/approximate-lower-bound-arguments/), May 2024, Eurocrypt'24 by _Pyrros Chaidos, Prof Aggelos Kiayias, Leonid Reyzin, Anatoliy Zinovyev_.
 
-Implementation of _Approximate Lower Bound Arguments_ from the [paper](https://iohk.io/en/research/library/papers/approximate-lower-bound-arguments/) published by IOG Research:
+Alba is a generic protocol to prove succinctly a lower bound of the size of a *potentially* weighted set.
+Say we have a set $S_p$ of size $|S_p| \geq n_p$, and a lower bound $n_f < n_p$ of it we want to prove.
```

Comment:
We should clarify to the reader why we don't simply use `n_p = |S_p|`, i.e. ,why not use two parameters instead of three.